src/main/java/roomescape/controller/reservation/ReservationController.java

```diff
@@ -20,32 +21,39 @@ public class ReservationController {
 
     private final ReservationService reservationService;
 
-    public ReservationController(ReservationService reservationService) {
+    public ReservationController(final ReservationService reservationService) {
         this.reservationService = reservationService;
     }
 
     @GetMapping
-    public ResponseEntity<List<ReservationResponse>> getReservations() {
-        List<ReservationResponse> reservations = reservationService.getReservations();
-        return ResponseEntity.ok(reservations);
+    public List<ReservationResponse> getReservations() {
+        return reservationService.getReservations();
     }
 
     @PostMapping
     public ResponseEntity<ReservationResponse> addReservation(@RequestBody final ReservationRequest request) {
-        ReservationResponse reservation = reservationService.addReservation(request);
+        final ReservationResponse reservation = reservationService.addReservation(request);
+        final URI uri = UriComponentsBuilder.fromPath("/reservations/{id}")
+                .buildAndExpand(reservation.id())
+                .toUri();
 
-        return ResponseEntity
-                .status(HttpStatus.OK)
-                .location(URI.create("/reservations/" + reservation.id()))
+        return ResponseEntity.created(uri)
                 .body(reservation);
     }
 
     @DeleteMapping("/{id}")
     public ResponseEntity<Void> deleteReservationsData(@PathVariable("id") final Long id) {
-        int deletedCount = reservationService.deleteReservation(id);
+        final int deletedCount = reservationService.deleteReservation(id);
         if (deletedCount == 0) {
-            return ResponseEntity.notFound().build();
+            return ResponseEntity.notFound()
+                    .build();
         }
```

Comment:
데이터를 삭제 개수가 0개일 때, status not found를 응답하는 상황이 상당히 어색해보입니다.
```
HTTP status code 404 means "Page Not Found". 
This means that the request you sent was received by the server, 
but it could not find the page you were looking for.
```

https://umbraco.com/knowledge-base/http-status-codes/#404-not-found